packages/webkit2gtk_4.rb

```diff
@@ -136,38 +133,45 @@ def self.patch
   end
 
   def self.build
-    # This builds webkit2gtk4 (which uses gtk3)
-    FileUtils.mkdir_p 'builddir'
+    # This builds webkit2gtk4 (which uses gtk3 and libsoup2)
     @workdir = `pwd`.chomp
-    Dir.chdir 'builddir' do
-      # Bubblewrap sandbox breaks on epiphany with
-      # bwrap: Can't make symlink at /var/run: File exists
-      # LDFLAGS from debian: -Wl,--no-keep-memory
-      unless File.file?('build.ninja')
-        @arch_linker_flags = ARCH == 'x86_64' ? '' : '-Wl,--no-keep-memory'
-        system "CREW_LINKER_FLAGS='#{@arch_linker_flags}' CC='#{@workdir}/bin/gcc' CXX='#{@workdir}/bin/g++' cmake \
-            -G Ninja \
+    # Bubblewrap sandbox breaks on epiphany with
+    # bwrap: Can't make symlink at /var/run: File exists
+    # LDFLAGS from debian: -Wl,--no-keep-memory
+    unless File.file?('build.ninja')
+      @arch_linker_flags = ARCH == 'x86_64' ? '' : '-Wl,--no-keep-memory'
```

Comment:
Conditional not needed?

Reply:
Might need it in case armv7l builds start working...